package.json

```diff
@@ -15,13 +15,13 @@
         "prerelease": "npm run validate",
         "release": "standard-version --commit-all",
         "postrelease": "git push && git push --follow-tags && npm run build && npm publish",
-        "test": "jest src",
-        "test:bundle": "jest bundle-tests/**",
+        "test": "vitest src",
+        "test:bundle": "vitest bundle-tests/**",
```

Comment:
whats the difference between these test suites? is one really slow?

I would recommend just running all tests in `test` unless the bundle tests are really slow

Reply:
ah yeah that would be annoying then. LGTM